### PR TITLE
Add: Policy toggles to Firewall rule tables

### PR DESCRIPTION
**Pull request by Jskobos**
## Description

You should be able to test this using dev services. There are some edge cases to work out in a future PR (specifically, we're defaulting outbound_policy to ACCEPT for new firewalls, but then if someone adds an outbound rule set to ACCEPT we might want to auto-toggle outbound_policy to DROP.

### CODE REVIEW

File: packages/manager/src/features/Firewalls/FirewallDetail/Rules/FirewallRuleTable.tsx [hunk 1]
```diff
@@ -78,6 +85,12 @@ const useStyles = makeStyles((theme: Theme) => ({
   dragging: {
     display: 'table',
   },
+  policyText: {
+    textAlign: 'right',
+  },
+  policySelect: {
+    paddingLeft: 4,
```
Comment (johnwcallahan) [inline packages/manager/src/features/Firewalls/FirewallDetail/Rules/FirewallRuleTable.tsx:92]: Minor: to my eye this needs to be 3px to be aligned with the actions above .

**Review by johnwcallahan — APPROVED** [review]
Functionality works well, design ongoing.

**Review by tiffwong — APPROVED** [review]

File: packages/manager/src/features/Firewalls/FirewallDetail/Rules/FirewallRuleTable.tsx [hunk 2]
```diff
@@ -86,6 +86,13 @@ const useStyles = makeStyles((theme: Theme) => ({
   dragging: {
     display: 'table',
   },
+  footer: {
+    '&:before': {
+      display: 'block',
+      content: '',
```
Comment (johnwcallahan) [inline packages/manager/src/features/Firewalls/FirewallDetail/Rules/FirewallRuleTable.tsx:92]: @tiffwong this needs to be `content: '""',` (trips me up every time).

File: packages/manager/src/features/Firewalls/FirewallDetail/Rules/FirewallRuleTable.tsx [hunk 3]
```diff
@@ -336,6 +384,63 @@ const FirewallRuleTableRow: React.FC<FirewallRuleTableRowProps> = React.memo(
...
+  const xsDown = useMediaQuery(theme.breakpoints.down('xs'));
+  const mdDown = useMediaQuery(theme.breakpoints.down('md'));
+  const colSpan = xsDown ? 1 : mdDown ? 4 : 5;
+
+  const helperText = mdDown ? (
+    <strong>{capitalize(category)} policy:</strong>
+  ) : (
+    <span>
+      <strong>Default {category} policy.</strong> This policy applies to any
+      traffic not covered by the inbound rules listed above.
```
Comment (johnwcallahan) [inline packages/manager/src/features/Firewalls/FirewallDetail/Rules/FirewallRuleTable.tsx:414]: "covered by the {category} rules listed above."

**Review by johnwcallahan — CHANGES_REQUESTED** [review]
Two small things.

**Review by johnwcallahan — APPROVED** [review]
